service/src/test/java/uk/gov/mca/beacons/api/export/mappers/ExportMapperUnitTest.java

```diff
@@ -363,7 +363,7 @@ public void toLabelDTO_whenTheGivenRegistrationHasNoLastModifiedDate_shouldMapPr
   @Test
   public void toLabelDTO_whenTheGivenRegistrationIsValid_shouldMapToLabelDTO() {
     Registration registration = new Registration();
-    Beacon beacon = new Beacon();
+    Beacon beacon = Beacon.builder().build();
```

Comment:
Same here, makes more sense to just have the empty constructor than creating a builder that isn't used

Reply:
Maybe you need to add

  @NoArgsConstructor
  @AllArgsConstructor
  
  To the class if the constructors are missing